fmriprep/workflows/epi.py

```diff
@@ -73,53 +73,56 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
 
     # Build workflow
     workflow = pe.Workflow(name=name)
-    inputnode = pe.Node(niu.IdentityInterface(fields=['epi',
-                                                      't1_preproc',
-                                                      't1_brain',
-                                                      't1_mask',
-                                                      't1_seg',
-                                                      't1_tpms',
-                                                      't1_2_mni_forward_transform',
-                                                      'subjects_dir',
-                                                      'subject_id',
-                                                      'fs_2_t1_transform',
-                                                      't1w']),
-                        name='inputnode')
+    inputnode = pe.Node(niu.IdentityInterface(
+        fields=['epi', 't1_preproc', 't1_brain', 't1_mask', 't1_seg', 't1_tpms',
+                't1_2_mni_forward_transform', 'subjects_dir', 'subject_id', 'fs_2_t1_transform']),
+        name='inputnode')
     inputnode.inputs.epi = bold_file
 
+    outputnode = pe.Node(niu.IdentityInterface(
+        fields=['epi_t1', 'epi_mask_t1', 'epi_mni', 'epi_mask_mni', 'confounds', 'surfaces']),
+        name='outputnode')
+
+    func_reports_wf = init_func_reports_wf(reportlets_dir=reportlets_dir,
+                                           freesurfer=freesurfer)
+
+    func_derivatives_wf = init_func_derivatives_wf(output_dir=output_dir,
+                                                   output_spaces=output_spaces,
+                                                   freesurfer=freesurfer)
+
+    workflow.connect([
+        (inputnode, func_reports_wf, [(('epi', _first), 'inputnode.source_file')]),
```

Comment:
Any derivatives/reports used [this construct](https://github.com/poldracklab/fmriprep/pull/482/files#diff-4c51aa6368169bddb20f09d1f776098dL440), so I just copied it. Looking at it, I think it's unnecessary because `'epi'` is always a single path, so I'll go ahead and remove it.